packages/core-ethereum/src/index.ts

```diff
@@ -295,25 +233,11 @@ export default class HoprEthereum implements HoprCoreConnector {
       network,
       hoprChannels,
       hoprToken,
-      { debug: options?.debug || false },
+      options?.debug || false,
```

Comment:
I think `debug ?? false` fits better here

Reply:
I just unwrapped the options. Didn't change the syntax.